packages/data-point/src/DataPoint.js

```diff
@@ -0,0 +1,125 @@
+const { createReducer } = require("./create-reducer");
```

Comment:
this file has several async functions, but they don't use `await` - do they still need to be async?